homeassistant/components/logger/__init__.py

```diff
@@ -70,6 +74,9 @@ def set_log_levels(logpoints):
     if LOGGER_LOGS in config[DOMAIN]:
         set_log_levels(config[DOMAIN][LOGGER_LOGS])
 
+    if config[DOMAIN][LOGGER_FAST]:
+        hass.bus.async_fire(EVENT_ENABLE_FAST_LOGGING)
```

Comment:
Not sure if fast logging should be enabled by an event, what would be a better way?